api/capi/meson.build

```diff
@@ -94,6 +81,43 @@ nnstreamer_capi_dep = declare_dependency(link_with: nnstreamer_capi_lib,
   ]
 )
 
+# New single-shot c-api
+capi_single_new_main = []
+capi_single_new_main += join_paths(meson.current_source_dir(), 'src', 'nnstreamer-capi-util.c')
+capi_single_new_main += join_paths(meson.current_source_dir(), 'src', 'nnstreamer-capi-single-new.c')
+capi_single_new_main += join_paths(meson.current_source_dir(), 'src', 'tensor_filter_single.c')
+
+nnstreamer_capi_single_new_lib = shared_library ('capi-nnstreamer-single-new',
+  capi_single_new_main,
+  dependencies: capi_deps,
+  include_directories: [
+    inc,
+    nninc
```

Comment:
how about declaring a variable for include_directories (because it's repeated)? like `capi_inc`

Reply:
done